base/applications/taskmgr/lang/nl-NL.rc

```diff
@@ -406,7 +406,7 @@ BEGIN
     IDS_TAB_IOOTHERBYTES "I/O: overige bytes"
     IDS_TAB_COMMANDLINE "Command Line"
     IDS_MENU_SELECTCOLUMNS "&Kolommen selecteren..."
-    IDS_MENU_16BITTASK "16-&bits taken weergeven" //FIXME: accelerator collision &b with ID_OPTIONS_MINIMIZEONUSE
+    IDS_MENU_16BITTASK "&16-bits taken weergeven"
```

Comment:
If using "t" as the accelerator, would this collide with something else?

Reply:
I cannot say, but I am 100% sure, that the &1 won't collide with anything else in that menu (some other languages do use &1 as well)

Reply:
My concern is _also_ about having the keyboard accelerator on a "thin" letter.
See https://learn.microsoft.com/en-us/previous-versions/windows/desktop/dnacc/guidelines-for-keyboard-user-interface-design#selecting-access-keys
> Make access keys discoverable.
>
>- Access keys must be easy for the user to find. Choose letters for access keys as close to the beginning of the label as possible for example, Settings. The following are good letter choices:
    * Letters at the beginning of the first or second word of the label.
    * A distinctive consonant or a vowel in the label.
    * _**Letters with wide widths, such as w, m, and capital letters.**_
>
>- Avoid using letters that are difficult to see when they are assigned to access keys. These letters include:
    * Letters with descenders, such as p, g, q, or y.
    * Letters next to a letter with a descender.
    * _**Letters that are only one pixel wide, such as i or l.**_

Reply:
Wise words from MS. Wise words from you.
Nevertheless I do think the &1 is not as bad as &i. It is considerably less "thin". I do think the result is visually appealing, see the pic:
![image](https://github.com/reactos/reactos/assets/33393466/deac8094-987a-445d-bbc1-26e3a33cad48)

&1 is also used in some other langs, so I don't see using it here as well as a big no-no.
Best ofc would be knowing what MS actually uses here for nl-NL, but I don't have a netherlandish Windows version to find that out for sure.
So I decided to stick with &1 for now **to solve the existing collision at least**, that we had here.
To address your concern I decided to add a //FIXME: Please double-check this accelerator
at that line so that a native netherland-er can cover the details later.

Reply:
OK no problem then, if we already did that in other languages. But this also depends on the user GUI font: if it doesn't print the 1 with an extra horizontal bar at the bottom (the serif) and thus, its width be further reduced then the keyboard accelerator symbol may not be seen greatly.
Also one may say that from some viewing distance, the keyboard accelerator bar may be confounded with the serif.